constructor/header.sh

```diff
@@ -391,14 +406,19 @@ cd "$PREFIX"
 unset PYTHON_SYSCONFIGDATA_NAME _CONDA_PYTHON_SYSCONFIGDATA_NAME
 
 # the first binary payload: the standalone conda executable
-CONDA_EXEC="$PREFIX/conda.exe"
+CONDA_EXEC="$PREFIX/_conda"
 extract_range "${boundary0}" "${boundary1}" > "$CONDA_EXEC"
 chmod +x "$CONDA_EXEC"
 
 export TMP_BACKUP="${TMP:-}"
 export TMP="$PREFIX/install_tmp"
 mkdir -p "$TMP"
 
+# Create $PREFIX/.nonadmin if the installation didn't require superuser permissions
+if [ "$(id -u)" -ne 0 ]; then
+    touch "$PREFIX/.nonadmin"
+fi
```

Comment:
I am following the pattern on Windows. We can also remove the `.nonadmin` business here because `menuinst` will try to create it too.

This will only create the `.nonadmin` file if the user is _not_ root. On Unix systems, I would hope most folks are running the installer with a mere `bash` and not `sudo bash`, though.

Reply:
I think for Windows, this is actually a bigger problem since it will ask for elevated privileges when it doesn't encounter a file. Not a problem for this PR though.